src/pages/workspaces/workspace/Workflows.js

```diff
@@ -248,20 +308,21 @@ const FindWorkflowModal = ajaxCaller(class FindWorkflowModal extends Component {
 
   async exportMethod() {
     const { namespace, name } = this.props
-    const { selectedWorkflow } = this.state
+    const { selectedWorkflow, workflowRename, methodAjax, config } = this.state
 
     this.setState({ exporting: true })
 
     try {
-      const methodAjax = Ajax().Methods.method(selectedWorkflow.namespace, selectedWorkflow.name, selectedWorkflow.snapshotId)
-
-      const config = _.maxBy('snapshotId', await methodAjax.configs())
+      const newConfig = (config && workflowRename !== selectedWorkflow.name) ?
+        _.merge(config, { name: workflowRename, payloadObject: { name: workflowRename } }) : undefined
```

Comment:
This looks like it will result in `undefined` if the name is edited, which would mean other information from the config would be lost.

Also, since we're already maintaining `workflowRename` as a valid value throughout, we could probably remove the conditional logic here and just always write the new value (which might be a noop if it wasn't edited)